OpenFilesScreen.c

```diff
@@ -191,6 +197,23 @@ static OpenFiles_ProcessData* OpenFilesScreen_getProcessData(pid_t pid) {
       pdata->error = WEXITSTATUS(wstatus);
    }
 
+   /* On linux, `lsof -o -F` omits SIZE, so add it back. */
+   /* On macOS, `lsof -o -F` includes SIZE, but overwrite it for consistency. */
+   fdata = pdata->files;
+   size_t sizeIndex = getIndexForType('s');
+   size_t filenameIndex = getIndexForType('n');
+   while (fdata) {
+     item = &(fdata->data);
+     char* filename = item->data[filenameIndex];
+     struct stat st;
+     if (stat(filename, &st) == 0) {
+       char sizeChar[11]; /* 10 + 1 (NULL) */
+       xSnprintf(sizeChar, sizeof(sizeChar), "%10lld", (long long)st.st_size); /* st.st_size is long long on macOS, long on linux */
```

Comment:
`%10lld` (uint64_t) can be larger than 10 characters.

Reply:
20 characters seems like too much for common terminal widths.  16 characters is enough for ext4’s max file size of 16TB. Or should I calculate what width is needed?

Reply:
Make it 20. 'cause everything else can be used to overrun the buffer. Safe coding is always preferred.